app/models/agents/http_status_agent.rb

```diff
@@ -12,6 +12,7 @@ class HttpStatusAgent < Agent
 
     form_configurable :url
     form_configurable :disable_redirect_follow, type: :array, values: ['true', 'false']
+    form_configurable :changes_only, type: :array, values: ['true', 'false']
```

Comment:
Maybe we should add `"changes_only" => false` in the default options to keep the old defaults for new agents?